Project/Gem/Source/Vision/IdealVisionSystem.cpp

```diff
@@ -183,7 +192,7 @@ namespace ROS2::Demo
         IdealVisionSystemConfiguration::Reflect(context);
         if (AZ::SerializeContext* serialize = azrtti_cast<AZ::SerializeContext*>(context))
         {
-            serialize->Class<IdealVisionSystem, ROS2SensorComponent>()->Version(1)->Field(
+            serialize->Class<IdealVisionSystem, ROS2SensorComponentBase>()->Version(1)->Field(
```

Comment:
Technically it does not matter much, but I have updated version numbers for other sensors, after adjusting them to new base.